src/@types/CommandContext.d.ts

```diff
@@ -0,0 +1,103 @@
+import type {
+	Client,
+	CommandInteraction,
+	CommandInteractionOption,
+	DMChannel,
+	Guild,
+	GuildMember,
+	GuildTextBasedChannel,
+	InteractionReplyOptions,
+	Message,
+	MessageReplyOptions,
+	User,
+} from 'discord.js';
+
+declare global {
+	/** Information relevant to a command invocation. */
+	interface CommandContext {
+		/** The command invocation interaction. */
+		readonly interaction: CommandInteraction;
+
+		/** Our own signed-in Discord client. */
+		readonly client: Client<true>;
+
+		/** The place to submit informative debug messages. */
+		readonly logger: Console; // TODO: More extensible logger than `console`?
```

Comment:
We should start a thread about this on Discord.

Reply:
I'm not sure when to bring it up, tbh. It's not really an issue until we start running the bot full-time and need better logging and reporting tools. At the very least, there's code in here that's ready to take a new logger when/if we decide to do that.

Reply:
But yes, we should discuss this at some point, either now or later.